lib/bolt/cli.rb

```diff
@@ -493,6 +495,15 @@ def execute(options)
         exit!
       end
 
+      @analytics = Bolt::Analytics.build_client
```

Comment:
This won't catch help, right. Should it?

Reply:
I left that out because it was sort of unclear what that means. We have both `bolt help task` and `bolt task --help`. Would those both just be "help" or would we want to try to distinguish what they were asking for help about?